packages/turf-transform-rotate/test.js

```diff
@@ -0,0 +1,66 @@
+const test = require('tape');
+const path = require('path');
+const fs = require('fs');
+const load = require('load-json-file');
+const write = require('write-json-file');
+const centroid = require('@turf/centroid');
+const helpers = require('@turf/helpers');
+const featureCollection = helpers.featureCollection;
+const point = helpers.point;
+const lineString = helpers.lineString;
+const rotate = require('./');
+
+const directories = {
+    in: path.join(__dirname, 'test', 'in') + path.sep,
+    out: path.join(__dirname, 'test', 'out') + path.sep
+};
+
+const fixtures = fs.readdirSync(directories.in).map(filename => {
+    return {
+        filename,
+        name: path.parse(filename).name,
+        geojson: load.sync(directories.in + filename)
+    };
+});
+
+test('rotate', t => {
+    for (const {filename, name, geojson}  of fixtures) {
+        let {angle, pivot} = geojson.properties || {};
+
+        const rotated = rotate(geojson, angle, pivot);
+
+        // style result
+        if (rotated.geometry.type === 'Point' || rotated.geometry.type === 'MultiPoint') {
+            rotated.properties['marker-color'] = '#F00';
+            rotated.properties['marker-symbol'] = 'star';
+        } else {
+            rotated.properties["stroke"] = "#F00";
+            rotated.properties["stroke-width"] = 4;
+        }
+        if (pivot) {
```

Comment:
This type of logic makes the tests confusing.

Simplified this as a single function.

```javascript
function makePivot(pivot, geojson) {
    if (!pivot) return centroid(geojson);
    return point(getCoord(pivot), {'marker-symbol': 'circle'});
}
```